### PR TITLE
Update login message to avoid duplicate brand name

### PR DESCRIPTION
**Pull request by nixocio**
Returned brandName already contain the name brand Ansible. Update to
avoid name brand repetition.

Also update brandName on document title.

See: https://github.com/ansible/awx/issues/9126

### CODE REVIEW

File: awx/ui_next/src/screens/Login/Login.jsx [hunk 1]
```diff
@@ -147,7 +147,7 @@ function AWXLogin({ alt, isAuthenticated }) {
     <Login header={Header} footer={Footer}>
       <LoginMainHeader
         data-cy="login-header"
-        title={brandName ? t`Welcome to Ansible ${brandName}!` : ''}
```
Comment (nixocio) [inline awx/ui_next/src/screens/Login/Login.jsx:150]: @jakemcdermott, I also looked for other parts of the code that we are using `brandName` and the only other one similar case that I found was on the `document.title` - updated accordingly.

**Review by AlexSCorey — APPROVED** [review]
Looks good.  

We now have some dead strings in the `.po` files like this [one](https://github.com/ansible/awx/pull/10179/files#diff-53a6bf78d750453126d950e9fdf791cd43c9c40777d12a870310665d47990e3aR9524) but I have a meeting on the calendar with Christian to go through them to clean up all the dead strings before we push for translation services.

**Comment by nixocio** [comment]
> We now have some dead strings in the .po

@AlexSCorey, what is the root cause of this issue? - I updated the strings using the `npm run  extract-strings`.

File: awx/ui_next/src/screens/Login/Login.jsx (hunk 1 above)
Comment (jakemcdermott) [reply awx/ui_next/src/screens/Login/Login.jsx:150]: sgtm

**Comment by AlexSCorey** [comment]
The dead strings are related to when you update existing strings, which are now on a new line in the file.  Ling ui sees them as new string entirely instead of updated existing strings.  It's not a big deal since there is a cli command to remove them.  Also, I believe we won't have this problem very soon because it looks like ling ui is going to fix the issue that forces us to run that `extract-strings` so frequently https://github.com/lingui/js-lingui/issues/1003

**Review by keithjgrant — APPROVED** [review]

**Review by tiagodread — APPROVED** [review]

**Pull request merged**